src/plone/api/content.py

```diff
@@ -16,18 +21,24 @@
 from zope.interface import Interface
 from zope.interface import providedBy
 
-import pkg_resources
 import random
 import transaction
 
 try:
-    pkg_resources.get_distribution('Products.Archetypes')
-except pkg_resources.DistributionNotFound:
+    get_distribution('Products.Archetypes')
+except DistributionNotFound:
     class IBaseObject(Interface):
```

Comment:
Missed this somehow, sorry.

This won't happen, because this IBaseObject is a different class than the one in Products.Archetypes.
It is probably clearer when you look at the .\__identifier__ (or the dotted name): `Products.Archetypes.interfaces.base.IBaseObject` or `plone.api.content.IBaseObject`.

'obj' may provide 'Products.Archetypes.interfaces.base.IBaseObject', but will never 'plone.api.content.IBaseObject'.
Unless, of course, people start doing 'zope.interface.alsoProvides(obj, plone.api.content.IBaseObject)'. Please do not do this :)